pallets/subtensor/src/tests/staking2.rs

```diff
@@ -817,9 +844,9 @@ fn test_stake_fee_calculation() {
         let total_hotkey_alpha = AlphaCurrency::from(100_000_000_000);
         let tao_in = TaoCurrency::from(100_000_000_000); // 100 TAO
         let reciprocal_price = 2; // 1 / price
-        let stake_amount = 100_000_000_000_u64;
+        let stake_amount = TaoCurrency::from(100_000_000_000);
 
-        let default_fee = 0; // FIXME: DefaultStakingFee is deprecated
+        let default_fee = TaoCurrency::ZERO; // FIXME: DefaultStakingFee is deprecated
```

Comment:
Is this comment still valid?

```suggestion
        let default_fee = TaoCurrency::ZERO;
```

Reply:
The test still relies on this value. We should probably refactor the test.